docs/src/pages/customization/styled/UsingOptions.js

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {
+  experimentalStyled as styled,
+  createTheme,
+  ThemeProvider,
+} from '@material-ui/core/styles';
+
+const customTheme = createTheme({
+  components: {
+    MyThemeComponent: {
+      styleOverrides: {
+        root: {
+          color: 'darkslategray',
+        },
+        primary: {
+          color: 'darkblue',
+        },
+        secondary: {
+          color: 'darkred',
+          backgroundColor: 'pink',
+        },
+      },
+      variants: [
+        {
+          props: { variant: 'dashed', color: 'primary' },
+          style: {
+            border: '1px dashed darkblue',
+          },
+        },
+        {
+          props: { variant: 'dashed', color: 'secondary' },
+          style: {
+            border: '1px dashed darkred',
+          },
+        },
+      ],
+    },
+  },
+});
+
+const MyThemeComponent = styled('div', {
+  // configure which props should be forwarded on dom
```

Comment:
```suggestion
  // Configure which props should be forwarded on DOM
```

Is "forwarded on DOM" correct?